tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -377,21 +407,32 @@ def init_full_results(self, full_results):
         )
         return full_results
 
-    def copy_es_data(self, elasticsearch, full_results):
+    def copy_es_data(self, elasticsearch):
         """
         Copy data from Internal ES (if exists) to the main ES
 
         Args:
             elasticsearch (obj): elasticsearch object (if exits)
-            full_results (obj): the full results object
+
+        Returns:
+            bool: True if data was copy to the main ES False otherwise
 
         """
         if elasticsearch:
             log.info("Copy all data from Internal ES to Main ES")
-            elasticsearch._copy(full_results.es)
-        # Adding this sleep between the copy and the analyzing of the results
-        # since sometimes the results of the read (just after write) are empty
-        time.sleep(10)
+            log.info("Dumping data from the Internal ES to tar ball file")
+            elasticsearch.dumping_all_data(self.full_log_path)
+            es_connection = self.backup_es
+            es_connection["host"] = es_connection.pop("server")
+            es_connection.pop("url")
+            if elasticsearch_load(self.main_es, self.full_log_path):
+                # Adding this sleep between the copy and the analyzing of the results
+                # since sometimes the results of the read (just after write) are empty
+                time.sleep(10)
+                return True
+            else:
+                log.warning("Can not upload data into the Main ES server")
```

Comment:
Same spelling fix here : Cannot instead of Can not

Reply:
fixed it